datafusion/datasource-parquet/src/file_format.rs

```diff
@@ -350,15 +352,17 @@ impl FileFormat for ParquetFormat {
             Some(time_unit) => Some(parse_coerce_int96_string(time_unit.as_str())?),
             None => None,
         };
-        let config_file_decryption_properties = &self.options.crypto.file_decryption;
+        #[cfg(feature = "parquet_encryption")]
```

Comment:
It would be great if there wa ssome way to encapsulate this somehow into a function or something that could be cfg'd rather than sprinking them around

For example, something like
```
#[cfg(..)]
fn add_crypto_properties(...)
```